docs/milestone-1.md

```diff
@@ -0,0 +1,35 @@
+# Milestone 1
```

Comment:
Better to not include report in the documentation=) I meant to put it as description  in your PR